fastchat/serve/model_worker.py

```diff
@@ -243,20 +244,47 @@ def generate_gate(self, params):
     def get_embeddings(self, params):
         try:
             tokenizer = self.tokenizer
-            input_ids = tokenizer.encode(params["input"], return_tensors="pt").to(
-                self.device
-            )
-            model_output = self.model(input_ids, output_hidden_states=True)
-            is_chatglm = "chatglm" in str(type(self.model)).lower()
-            if is_chatglm:
-                data = (model_output.hidden_states[-1].transpose(0, 1))[0]
+            is_vicuna = "llama" in str(type(self.model)) # vicuna support batch inference
```

Comment:
`is_vicuna` -> `is_llama`

Reply:
'llama' or 'LLama'?

Reply:
> 'llama' or 'LLama'?

It shows that `transformers.models.llama.modeling_llama.LlamaForCausalLM`